src/main/java/com/sams/demo/data/Question.java

```diff
@@ -0,0 +1,44 @@
+package com.sams.demo.data;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "Questions")
+public class Question {
+
+    @Id @GeneratedValue(strategy = GenerationType.AUTO)
+    private final int id;
+    private final String title;
+    private final String link;
+    private final String level;
+
```

Comment:
we need only default constructor without any implementation. Thus we can remove both constructors.
And please use hibernate annotations for all properties mapped to table properties. Use annotations attributes to indicate length, nullable... and so on. Each annotation on separate line